content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -31,52 +31,24 @@ but not production. Like all alpha features, this guide is subject to change.
 1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
 
 ## Prepare the guide environment
-
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a VM in one of the cloud provider
+1. Set the environment variables `"${VM_NAME}"`, `"${WORK_DIR}"`,`"${CLUSTER_NAME}"` , `"${SERVICE_ACCOUNT}"`,
+，`"${SERVICE_ACCOUNT}"` your cluster name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
     (e.g. `WORK_DIR="${HOME}"/vmintegration`).
 
     {{< text bash >}}
-    $ ISTIO_DIR="<the directory containing an unarchived version of Istio>"
+    $ VM_NAME="<the name of your vm instance you created>"
     $ CLUSTER_NAME="<the name of your cluster>"
     $ SERVICE_NAMESPACE="<the name of your service namespace>"
     $ WORK_DIR="<a certificate working directory>"
+    $ SERVICE_ACCOUNT="<name of the service account you want to create for k8s>"
     {{< /text >}}
 
 1. Create the `"${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"` working directories.
 
     {{< text bash >}}
     $ mkdir -p "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"
     {{< /text >}}
-
-## Create certificates for use with the virtual machine and Istio control plane
-
-{{< tip >}}
-This `Makefile` is limited to creating one virtual machine certificate per cluster. The Istio authors
-expect operators to read and understand this guide to formulate their own plans for creating and
-managing virtual machines. It is important for you to read and understand this `Makefile` for any
-deployment you place into production.
-{{< /tip >}}
-
-1. Execute the following commands to create certificates for use by Istio. See
-    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
-    for more details on configuring an external CA. The `NAME` variable is
-    used during certificate generation to uniquely identify clusters. The
-    `NAMESPACE` variable identifies the namespace where the virtual machine
-    connectivity is hosted.
-
-    {{< text bash >}}
-    $ cd "${WORK_DIR}"
-    $ make -f "${ISTIO_DIR}"/tools/certs/Makefile NAME="${CLUSTER_NAME}" NAMESPACE="${SERVICE_NAMESPACE}" "${CLUSTER_NAME}"-cacerts-selfSigned
-    {{< /text >}}
-
-1. Execute the following commands to create certificates for use on the virtual machine.
-
-    {{< text bash >}}
-    $ cd "${WORK_DIR}"
-    $ make -f "${ISTIO_DIR}"/tools/certs/Makefile NAME="${CLUSTER_NAME}" NAMESPACE="${SERVICE_NAMESPACE}" "${NAMESPACE}"-certs-selfSigned
-    {{< /text >}}
-
 ## Install the Istio control plane
 
 The Istio control plane must be installed with virtual machine integration enabled (`values.global.meshExpansion.enabled: true`).
```

Comment:
showing how to do this, as per 1.6 instructions is valuable. Most people want to just cut and past the documentation, and this requires thinking 👍

Reply:
add explanation of set the IstioOperator spec `values.global.meshExpansion.enabled: true`